### PR TITLE
[master] Add information card

### PR DESCRIPTION
**Pull request by FranckKe**
Add page category for information card
Add account creation information
Add tracer information

info_card

Todo:
- [x] Add content
- [x] Translate content
  - [x] En
  - [x] Es
- [x] Dump updated pages content (update seeds)
- [x] Mentions légales
- [x] Update "Project" Pages

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,92 +1,152 @@
...
+        <p><span style="font-size: 14px;">Ocean Plastic Tracker est co-administré par <span style="font-size: 14px;"><span style="font-size: 14px;"><a href="https://assoansel.jimdo.com/" rel="noopener noreferrer" target="_blank">Ansel</a></span></span> et par <a href="https://sea-mer.org" rel="noopener noreferrer" target="_blank">Sea-Mer</a>. </span><span style="font-size: 14px;">L'application est </span><a href="https://github.com/FranckKe/mersea" rel="noopener noreferrer" target="_blank"><span style="font-size: 14px;">open source</span></a><span style="font-size: 14px;"> et en constante évolution grâce aux travail de </span><a href="https://github.com/FranckKe/mersea/graphs/contributors" rel="noopener noreferrer" target="_blank"><span style="font-size: 14px;">développeurs bénévoles</span></a><span style="font-size: 14px;">. </span></p>
```
Comment (Qayou) [inline db/seeds.rb]: au~~x~~ travail

File: db/seeds.rb [hunk 2]
```diff
@@ -1,92 +1,152 @@
...
+        <p><span style="font-size: 14px;">Parmi les pollutions plastiques, certains déchets racontent une histoire et permettent de suivre leur propagation au gré des courants et des tempêtes. Ces déchets sélectionnés, appelé "traceurs", permettent de qualifier et de quantifier une pollution provenant de conteneurs perdus en mer ou de déversements quelconques.</span></p>
```
Comment (Qayou) [inline db/seeds.rb]: appelé**s** "traceurs"

File: db/seeds.rb [hunk 3]
```diff
@@ -95,17 +155,412 @@
...
+
+# En
+
+Page.create!(
+  name: 'Ocean plastic tracker',
+  language: 'en',
+  category: 'welcome',
+  slug: 'ocean-plastic-tracker-en',
+  content: <<~ABOUT
+  <p><span style="font-size: 14px;">Ocean Plastic Tracker is a crowdsourcing tool used to geolocate specific marine waste that comes for example from <span style="font-size: 14px;">lost </span>containers. This map make it possible to visualize and better understand ocean <span style="font-size: 14px;">waste</span> distribution on the coast. The project aims to raise awareness of marine pollution and gather data to take action against polluters.</span></p>
```
Comment (Qayou) [inline db/seeds.rb]: This map make**s** it possible

File: db/seeds.rb [hunk 4]
```diff
@@ -95,17 +158,418 @@
...
+# Es
+
+Page.create!(
+  name: 'Ocean plastic tracker',
+  language: 'es',
+  category: 'welcome',
+  slug: 'ocean-plastic-tracker-es',
+  content: <<~ABOUT
+  <p><span style="font-size: 14px;">Ocean Plastic Tracker es una herramienta de crowdsourcing utilizada para geolocalizar residuos marinos espec&iacute;ficos que provienen, por ejemplo, de contenedores perdidos. Este mapa hace posible visualizar y comprender mejor la distribuci&oacute;n de los desechos oce&aacute;nicos en la costa. El objetivo del proyecto es aumentar la conciencia sobre la contaminaci&oacute;n marina y recopilar datos para tomar medidas contra los contaminadores.</span></p>
+  <p><span style="font-size: 14px;">Ocean Plastic Tracker es coadministrado por Ansel y Sea-Mer. La aplicaci&oacute;n es de c&oacute;digo abierto y est&aacute; en constante evoluci&oacute;n gracias al trabajo de desarrolladores voluntarios.</span></p>
```
Comment (Qayou) [inline db/seeds.rb]: Ansel and Sea-Mer links

Comment (Qayou) [inline db/seeds.rb]: GitHub contributors link

File: db/seeds.rb [hunk 5]
```diff
@@ -95,17 +158,418 @@
...
+    <p>If you see a bird in difficulty or injured, contact a <a href="http://www.ufcs.fr" rel="noopener noreferrer" target="_blank">UFCS</a> center or the nearest <a href="https://www.lpo.fr" rel="noopener noreferrer" target="_blank">LPO center</a>. They will advise you on the steps to follow. </p>
+    <h2>I observe marine litter and I get to know them</h2>
+    <p>Not all marine litter is alike. Some of them deserve special attention as they can come from far away or be unusual. Passionate collectors keep a collection of their most unique waste.</p>
+    <p>To dive into these mysteries and know the origin of the waste you collect, <a href="http://dechets-marins.ovh" rel="noopener noreferrer" target="_blank">Wiki-déchet</a> can help you</p>
+    <p>Enjoy beach cleaning and do not forget: When the beach is clean, it is most likely that the waste is in the water, or that they were collected!</p>
```
Comment (Qayou) [inline db/seeds.rb]: not forget: ~~W~~**w**hen the

File: db/seeds.rb [hunk 6]
```diff
@@ -95,17 +158,418 @@
...
+    <p>If you see a bird in difficulty or injured, contact a <a href="http://www.ufcs.fr" rel="noopener noreferrer" target="_blank">UFCS</a> center or the nearest <a href="https://www.lpo.fr" rel="noopener noreferrer" target="_blank">LPO center</a>. They will advise you on the steps to follow. </p>
+    <h2>I observe marine litter and I get to know them</h2>
+    <p>Not all marine litter is alike. Some of them deserve special attention as they can come from far away or be unusual. Passionate collectors keep a collection of their most unique waste.</p>
+    <p>To dive into these mysteries and know the origin of the waste you collect, <a href="http://dechets-marins.ovh" rel="noopener noreferrer" target="_blank">Wiki-déchet</a> can help you</p>
```
Comment (Qayou) [inline db/seeds.rb]: Need a point at the end of the sentence.

File: db/seeds.rb [hunk 7]
```diff
@@ -95,17 +158,418 @@
...
+    </ul>
+    <h2>Cargo and ship live tracking</h2>
+    <ul>
+        <li><a href="https://www.marinetraffic.com/en/ais/home/centerx:-4.3/centery:48.1/zoom:7" rel="noopener noreferrer" target="_blank">Marine Traffic</a></li>
+        <li><a href="https://www.vesselfinder.com/" rel="noopener noreferrer" target="_blank">Vessel Finder</a></li>
+    </ul>
+LINKS
+)
+
+Page.create!(
```
Comment (Qayou) [inline db/seeds.rb]: The paragraphs on this page do not have the same font size.

File: db/seeds.rb [hunk 8]
```diff
@@ -1,92 +1,155 @@
...
+    <h2>Développeurs</h2>
+    <p><span style="font-size: 14px;">La première version d'Ocean Plastic Tracker a vu le jour lors du Ocean Hackathon de Brest en octobre 2016. L'application web open source a été développé en 48 heures, par une équipe de quatre développeurs : Franck Kerbiriou, Marc Douchement, <span style="font-size: 14px;">Pierre Charpentier </span>et Caroline Geiser. Fiers de leurs travail et persuadés de son utilité, ils poursuivent aujourd'hui leurs efforts pour rendre l'application plus fiable, plus pratique et plus conviviale, tant pour les administrateurs que pour les observateurs. Une version 2, entièrement re-développé, à été mise en ligne en Avril 2019 et continue d'être amélioré. Le développement est ouvert à tous, rendez vous sur Github ou contactez-nous si vous êtes intéressé !</span>
+    </p>
+    <p><span style="font-size: 14px;">contact @ dev.oceanplastictracker.com, </span><a href="https://github.com/FranckKe/mersea" rel="noopener noreferrer" target="_blank"><span style="font-size: 14px;">Mersea sur Github</span></a></p>
```
[the remark this answers is not in the document]
Comment (FranckKe) [reply db/seeds.rb]: C'est pour essayer d'eviter le spam par robot crawling. Il n'y a pas vraiment de solutions miracles.

**Comment by Qayou** [comment]
LGTM (I have not checked the Spanish tho)

File: db/seeds.rb [hunk 9]
```diff
@@ -49,12 +49,12 @@
...
     <h2>Les impacts de la pollution plastique</h2>
-    <p><span style="font-size: 14px;">8 millions de tonnes de plastiques sont déversés dans les océans chaque année. Dommages sur les habitants, perturbateurs endocriniens, blessures, décès d'animaux marins par ingestion ou étranglement, ces pollutions massives entraînent des pertes économiques majeures (baisse du tourisme, impacts sur les pêches&hellip;).</span></p>
+    <p><span style="font-size: 14px;">8 millions tons de plastiques sont déversées dans les océans chaque année. Dommages sur les habitants, perturbateurs endocriniens, blessures, décès d'animaux marins par ingestion ou étranglement, ces pollutions massives entraînent des pertes économiques majeures (baisse du tourisme, impacts sur les pêches&hellip;).</span></p>
```
Comment (Qayou) [inline db/seeds.rb]: **tonnes**

**Pull request merged**